src/main/kotlin/br/unipar/plano/domain/reembolso/model/EnumEstados.kt

```diff
@@ -0,0 +1,31 @@
+package br.unipar.plano.domain.reembolso.model
+
+enum class  EnumEstados {
```

Comment:
Um enum é um novo tipo de dados, não precisa do prefixo `Enum`, bem como o nome do tipo é definido no singular já que ele não representa uma coleção.

```suggestion
enum class  Estado {
```